core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -1358,4 +1359,229 @@ class SQLPlanParserSuite extends BaseTestSuite {
+    DataWritingCommandExecParser.isWritingCmdExec(node.name) shouldBe true
+    val execInfo = DataWritingCommandExecParser.parseNode(node, new PluginTypeChecker(), 2)
+    f(execInfo)
+  }
+
+  test("DeltaLake Op AppendDataExecV1 and OverwriteByExpressionExecV1") {
+    // test AppendDataExecV1 with supported dataFormat
+    val nodeName = "AppendDataExecV1"
+    // scalastyle:off line.size.limit
+    val nodeDesc =
+      s"""|AppendDataExecV1 [num_affected_rows#18560L, num_inserted_rows#18561L], DeltaTableV2(org.apache.spark.sql.SparkSession@5aa5327e,abfss://abfs_path,Some(CatalogTable(
+          |Catalog: spark_catalog
+          |Database: adl
+          |Table: dbpp_tracker
+          |Owner: root
+          |Created Time: Wed Sep 15 16:47:47 UTC 2021
+          |Last Access: UNKNOWN
+          |Created By: Spark 3.1.1
+          |Type: EXTERNAL
+          |Provider: delta
+          |Table Properties: [bucketing_version=2, delta.lastCommitTimestamp=1631724453000, delta.lastUpdateVersion=0, delta.minReaderVersion=1, delta.minWriterVersion=2]
+          |Location: abfss://abfs_path
+          |Serde Library: org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe
+          |InputFormat: org.apache.hadoop.mapred.SequenceFileInputFormat
+          |OutputFormat: org.apache.hadoop.hive.ql.io.HiveSequenceFileOutputFormat
+          |Partition Provider: Catalog
+          |Schema: root
+          | |-- load_ts: string (nullable = true)
+          | |-- partition_dt: string (nullable = true)
+          | |-- job_name: string (nullable = true)
+          | |-- status: string (nullable = true)
+          | |-- batch_start_time: string (nullable = true)
+          | |-- batch_end_time: string (nullable = true)
+          | |-- app_id: string (nullable = true)
+          | |-- table_name: string (nullable = true)
+          | |-- data_dt: string (nullable = true)
+          |)),Some(spark_catalog.adl.dbpp_tracker),None,Map()), Project [from_unixtime(unix_timestamp(current_timestamp(), yyyy-MM-dd HH:mm:ss, Some(Etc/UTC), false), yyyy-MM-dd HH:mm:ss, Some(Etc/UTC)) AS load_ts#15200, 20240112 AS partition_dt#15201, load_master_brg_acct_sub_cim_rlnk AS job_name#15202, completed AS status#15203, from_unixtime(unix_timestamp(current_timestamp(), yyyy-MM-dd HH:mm:ss, Some(Etc/UTC), false), yyyy-MM-dd HH:mm:ss, Some(Etc/UTC)) AS batch_start_time#15204, from_unixtime(unix_timestamp(current_timestamp(), yyyy-MM-dd HH:mm:ss, Some(Etc/UTC), false), yyyy-MM-dd HH:mm:ss, Some(Etc/UTC)) AS batch_end_time#15205, ddsdmsp AS app_id#15206, master_brg_acct_sub_cim_rlnk AS table_name#15207, from_unixtime(unix_timestamp(current_timestamp(), yyyy-MM-dd HH:mm:ss, Some(Etc/UTC), false), yyyyMMdd, Some(Etc/UTC)) AS data_dt#15208], org.apache.spark.sql.execution.datasources.v2.DataSourceV2Strategy$$Lambda$$12118/1719387317@5a111bfa, com.databricks.sql.transaction.tahoe.catalog.WriteIntoDeltaBuilder$$anon$$1@24257336
```

Comment:
Can u use some test specific yet generic field names?

Reply:
I do not get that question

Reply:
Done!